orchagent/portsorch.cpp

```diff
@@ -1678,23 +1678,27 @@ bool PortsOrch::initPort(const string &alias, const set<int> &lane_set)
     return true;
 }
 
-void PortsOrch::deinitport(string alias, sai_object_id_t port_id)
+void PortsOrch::deinitport(const Port& p)
 {
     SWSS_LOG_ENTER();
 
-    Port p(alias, Port::PHY);
-    p.m_port_id = port_id;
+    string pAlias = p.m_alias;
+    sai_object_id_t pOID = p.m_port_id;
 
     /* remove port name map from counter table */
     RedisClient redisClient(m_counter_db.get());
-    redisClient.hdel(COUNTERS_PORT_NAME_MAP, alias);
+    redisClient.hdel(COUNTERS_PORT_NAME_MAP, pAlias);
+
+    // Destroy port's queue and PG flex stat counters
+    destroyQueueMapPerPort(p);
```

Comment:
This maps are are generated by FlexCounterOrch, should we  destroy them at similar place where we DEL flex counters?

Reply:
As we discussed, these are generated by PortsOrch but initiated by FlexCounterOrch on receiving the first flex counter group config. So, I feel this is the right place.